functions/PayrollProcessor.Functions/Features/Employees/EmployeesTrigger.cs

```diff
@@ -70,7 +70,7 @@ public class EmployeesTrigger
 
             var employee = option.IfNone(() => throw new Exception($"Could not find employee [{updateParams.Department}] [{id}]"));
 
-            employee.Status = updateParams.Status;
+            employee.Status = EmployeeDepartment.Find(updateParams.Status).CodeName;
```

Comment:
I thought about changing the `Status` property type to be `EmployeeStatus`, but didn't want to add that much strictness just yet. 🤷‍♂️